bots/task/github.py

```diff
@@ -325,6 +300,25 @@ def commits(self, branch='master', since=None):
                 yield commit
                 count += 1
 
+    def whitelist(self):
+        users = set()
+        teamId = self.teamIdFromName(TEAM_CONTRIBUTORS)
+        page = 1
+        count = 100
+        while count == 100:
+            data = self.get("/teams/{0}/members?page={1}&per_page={2}".format(teamId, page, count)) or []
+            users.update(user.get('login') for user in data)
+            count = len(data)
+            page += 1
+        return users
+
+    def teamIdFromName(self, name):
+        try:
+            return next(team for team in self.get("/orgs/cockpit-project/teams") or [] if team.get('name') == TEAM_CONTRIBUTORS).get('id')
+        except StopIteration:
```

Comment:
This is a really odd way of writing this search. How about this:

```py
for team in self.get("/orgs/cockpit-project/teams", []):
    if team.get("name") == TEAM_CONTRIBUTORS:
         return team["id"]
else:
    raise KeyError(".. not found")
```

Please also use consistent quoting and cause a `KeyError` if "id" is not present (the above does that).